seller-info-service/src/main/java/com/yandex/market/sellerinfoservice/repository/SellerRepository.java

```diff
@@ -2,12 +2,18 @@
 
 import com.yandex.market.sellerinfoservice.model.Seller;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
+import java.util.Optional;
 import java.util.UUID;
 
 public interface SellerRepository extends JpaRepository<Seller, Long> {
 
     boolean existsSellerByEmail(String email);
 
-    Seller getSellerByExternalId(UUID sellerExternalId);
+    @Query(value = """
+            FROM Seller seller
+            WHERE seller.externalId=:sellerExternalId
+            """)
+    Optional<Seller> getSellerByExternalId(UUID sellerExternalId);
```

Comment:
если обозвать метод так:
```suggestion
    Optional<Seller> findByExternalId(UUID sellerExternalId);
```
То запрос Query не нужен